content/index.md

```diff
@@ -26,7 +26,7 @@ action:
 work:
   - /work/coachhub/
   - /work/moztrap/
-  - /work/quarqrace/
+  - /work/quarqnet/
```

Comment:
@mirisuzanne Feel free to revert this if it was intentional -- I just thought the Quarqnet page (with case study) is more interesting to highlight than Quarqrace.

Reply:
that's fair. on the other hand: quarqrace is the more successful product. Guess we should make a case study for it.